src/include/distributed/version_compat.h

```diff
@@ -24,6 +24,45 @@
 #include "optimizer/optimizer.h"
 #endif
 
+#if (PG_VERSION_NUM >= PG_VERSION_13)
+#include "tcop/tcopprot.h"
+#endif
+
+#if PG_VERSION_NUM >= PG_VERSION_13
+#define heap_open(r, l) table_open(r, l)
+#define heap_openrv(r, l) table_openrv(r, l)
+#define heap_openrv_extended(r, l, m) table_openrv_extended(r, l, m)
+#define heap_close(r, l) table_close(r, l)
+#define lnext_compat(l, r) lnext(l, r)
+#define list_delete_cell_compat(l, c, p) list_delete_cell(l, c)
+#define pg_plan_query_compat(p, q, c, b) pg_plan_query(p, q, c, b)
+#define planner_compat(p, q, c, b) planner(p, q, c, b)
+#define standard_planner_compat(a, b, c, d) standard_planner(a, b, c, d)
+#define PortalDefineQuerySelectCompat(a, b, c, e, f) PortalDefineQuery(a, b, c, \
+																	   CMDTAG_SELECT, e, \
+																	   f)
+#define getOwnedSequencesCompat(a, b) getOwnedSequences(a)
+#define ExplainOnePlanCompat(a, b, c, d, e, f, g, h) ExplainOnePlan(a, b, c, d, e, f, g, \
+																	h)
+#define varoattno varattnosyn
+#define varnoold varnosyn
+#define Set_ptr_value(a, b) ((a)->ptr_value = (b))
+#define RangeTableEntryFromNSItem(a) ((a)->p_rte)
+#define QueryCompletionCompat QueryCompletion
+#else /* pre PG13 */
+#define lnext_compat(l, r) lnext(r)
+#define list_delete_cell_compat(l, c, p) list_delete_cell(l, c, p)
+#define pg_plan_query_compat(p, q, c, b) pg_plan_query(p, c, b)
+#define planner_compat(p, q, c, b) planner(p, c, b)
```

Comment:
Same as `ExplainOnePlanCompat` we should prefer new always-NULL parameters to be in the macro & not specified in parameter list